api/datastore/sql/sql.go

```diff
@@ -560,7 +560,8 @@ func (ds *sqlStore) RemoveRoute(ctx context.Context, appName, routePath string)
 }
 
 func (ds *sqlStore) GetRoute(ctx context.Context, appName, routePath string) (*models.Route, error) {
-	rSelectCondition := "%s WHERE app_name=? AND path=?"
+	fmt.Println("in GetRoute")
+	rSelectCondition := "%s WHERE app_name=? AND (path=? OR path='/*')"
```

Comment:
GetRoute looks like it evaluates one row. In this case, we should prefer the non '/*' match (exact match) if both are present.